### PR TITLE
Add file locking article

### PR DESCRIPTION
**Pull request by tomayac**
Changes proposed in this pull request:

- Add the article
- The review happened in this [Google doc](https://docs.google.com/document/d/1Sbqzz7-q2-MwHcWKeY1r54owLtaYkDrw3be6ks1ueps/edit#heading=h.u2q5uspa696r)

### CODE REVIEW

File: site/en/blog/new-dev-trial-for-multiple-readers-and-writers/index.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+layout: 'layouts/blog-post.njk'
+title: New dev trial for multiple readers and writers for FileSystemSyncAccessHandle and exclusive writer for FileSystemWritableFileStream
+subhead: >
+  Learn about API improvements when working with files in the browser.
```
Comment (tomayac) [inline site/en/blog/new-dev-trial-for-multiple-readers-and-writers/index.md:5]:
```suggestion
title: API Improvements for working with files in the browser
subhead: >
  New dev trial for multiple readers and writers for FileSystemSyncAccessHandle and exclusive writer for FileSystemWritableFileStream.
```

File: site/en/blog/new-dev-trial-for-multiple-readers-and-writers/index.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
+---
+layout: 'layouts/blog-post.njk'
+title: New dev trial for multiple readers and writers for FileSystemSyncAccessHandle and exclusive writer for FileSystemWritableFileStream
```
[the remark this answers is not in the document]
Comment (tomayac) [reply site/en/blog/new-dev-trial-for-multiple-readers-and-writers/index.md]: Hah, the solution to this was so simple. Thanks for pointing out the obvious. I was struggling…

**Review by rachelandrew — APPROVED** [review]

File: site/en/blog/new-dev-trial-for-multiple-readers-and-writers/index.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Mohammed71o) [reply site/en/blog/new-dev-trial-for-multiple-readers-and-writers/index.md]: a1d7025

https://pr-7476-static-dot-fcc-
staging.uc.r.appspot.com/

[the remark this answers is not in the document]
Comment (Mohammed71o) [reply site/en/blog/new-dev-trial-for-multiple-readers-and-writers/index.md]: https://pr-7476-static-dot-dcc-
staging.uc.r.appspot.com/blogne
w-dev-trial-for-multiple-readers-and-writers/. 


CC 4:

**Pull request merged**